### PR TITLE
Improve inset ticklabels. 

### PR DESCRIPTION
**Pull request by snim2**
Ticklabels are sometime strings, not floats, this commit avoids a ValueError.

### CODE REVIEW

File: bin/plot_krun_results [hunk 1]
```diff
@@ -677,7 +677,7 @@ def draw_page(is_interactive, executions, cycles_executions,
             if not inset_collides:
                 # Create and style the inset.
                 inset = add_inset_to_axis(fig, axis, rect)
-                inset.set_ylim([y_min, y_max])  # Same scale as larger subplot.
+                inset.set_ylim(axis.get_ylim())
```
Comment (ltratt) [inline bin/plot_krun_results]: Should this bit be part of another PR?

Comment (snim2) [reply bin/plot_krun_results]: Oh, possibly

**Comment by snim2** [comment]
Ready to squash?

File: bin/plot_krun_results [hunk 2]
```diff
@@ -677,7 +677,7 @@ def draw_page(is_interactive, executions, cycles_executions,
             if not inset_collides:
                 # Create and style the inset.
                 inset = add_inset_to_axis(fig, axis, rect)
-                inset.set_ylim([y_min, y_max])  # Same scale as larger subplot.
+                inset.set_ylim([y_min, y_max])
```
Comment (ltratt) [inline bin/plot_krun_results]: The undo hasn't worked, as it's killed the comment. Let's put the comment back in so that the PR has a tight focus.

**Comment by ltratt** [comment]
Yep, let's squash!

**Comment by snim2** [comment]
OK?

**Comment by ltratt** [comment]
Thanks!

**Review by ltratt — APPROVED** [review]
